migration/migration_pkg_test.go

```diff
@@ -64,3 +69,32 @@ func TestRelocateContractStorageRootKeys(t *testing.T) {
 		require.ErrorIs(t, db.ErrKeyNotFound, err)
 	}
 }
+
+func TestRecalculateBloomFilters(t *testing.T) {
+	testdb := pebble.NewMemTest()
+	t.Cleanup(func() {
+		require.NoError(t, testdb.Close())
+	})
+	chain := blockchain.New(testdb, utils.MAINNET, utils.NewNopZapLogger())
+	client, closeFn := feeder.NewTestClient(utils.MAINNET)
+	t.Cleanup(closeFn)
+	gw := adaptfeeder.New(client)
+
+	for i := uint64(0); i < 3; i++ {
+		b, err := gw.BlockByNumber(context.Background(), i)
+		require.NoError(t, err)
+		su, err := gw.StateUpdate(context.Background(), i)
+		require.NoError(t, err)
+
+		b.EventsBloom = nil
+		require.NoError(t, chain.Store(b, su, nil))
+	}
+
+	require.NoError(t, testdb.Update(recalculateBloomFilters))
+
+	for i := uint64(0); i < 3; i++ {
+		b, err := chain.BlockByNumber(i)
+		require.NoError(t, err)
+		require.NotNil(t, b.EventsBloom)
```

Comment:
Can we do something along the lines of `assert.Equal(t, core.EventsBloom(block.Receipts), b.EventsBloom)`?

Reply:
Yeap, much better.